### ISSUE TITLE
<s snippet does not expand in org-mode

### ISSUE THREAD

**Issue by siddharthshekar** [hlissner/doom-snippets#6]
When I try expanding `<s` in an `org-mode` buffer, it just ends up deleting the `<s` text and not replacing it with a `#+BEGIN_SRC...#+END_SRC` as expected. Expanding `src` works as expected, except it can't be undone (but that's a separate issue). Does the `<` in the key need to be escaped?

**Comment by hlissner** [comment]
Have you updated `doom-snippets`? https://github.com/hlissner/doom-snippets/commit/3ce0c49f56a02c0fb02a370582920c974f764b45 and https://github.com/hlissner/doom-snippets/commit/162713d691e6ef1257abe3a2273724763ae832a3 should've fixed this issue.

**Comment by siddharthshekar** [comment]
My bad! I didn't realize I wasn't on the latest version of doom snippets. It works as expected on updating.

It would be nice to be able to undo the snippet creation, but again, that's not this issue. I'll go ahead and close this one.

**Comment by hlissner** [comment]
> My bad! I didn't realize I wasn't on the latest version of doom snippets. 

No sweat!

> It would be nice to be able to undo the snippet creation

A snippet can be undone once it is finished (TAB'ed out of the last field, if any) or aborted.

If you use Doom Emacs, pressing `ESC` or `C-g` should abort the snippet (might have to press it twice; once to escape insert mode -- if you use evil -- then again to abort the snippet). Then pressing `u` (`M-x undo`) should undo the snippet.

In `<s`'s case, you'll have to press `u` twice, because the first time you press it, it will undo the abortion of the snippet. i.e. it will put your cursor back in the first field of the snippet.

The second time it will undo the snippet altogether.
